includes/bomberman.hpp

```diff
@@ -48,7 +48,7 @@
 
 #ifdef __APPLE__
```

Comment:
on pourrais pas faire un truc dans ce genre la ?
```suggestion
#ifdef __APPLE__ || __LINUX__
```

Reply:
nope, pour linux, on veux pas le homedir. 
```
#ifdef __APPLE__
	#define CONFIG_DIR				"/Library/Application Support/bomberman/configs/"
#elif
	#define CONFIG_DIR				"configs/"
#endif
```
`homeDir` est ajouté avant CONFIG_DIR, et du coup pour mac, on le mets dans `~/Library/Application Support/bomberman/configs/` et pour linux, on le laisse là où il était